package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www",
-    "test": "export NODE_ENV=test && mocha test"
+    "start": "DEBUG=test-app* node ./bin/www",
```

Comment:
The tutorial uses https://www.npmjs.com/package/debug to log the server listening events. So if you run the server without the debug logging enabled you won't see that the server is running.
https://github.com/TwilioDevEd/browser-calls-node/blob/next-gen/bin/www#L82